Added Bee Plush & Fixed Farwa Plush

jwc1015 opened the pull request:
<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You'll find a README and example file in .\html\changelogs\ for further instructions.

You can also find a template for adding your changelog directly to the PR description here: https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation
-->
:cl: jwc1015
rscadd: "Added a big space bee plushie! Also available in loadout!"
bugfix: "Enabled the Farwa plushie's handheld sprite."
/:cl:

jwc1015 committed:
Added Bee Plush & Fixed Farwa Plush

SierraKomodo reviewed (changes requested):
With the bee in your left hand, the left and right facing sprites are incorrect.

![dreamseeker_97Vzmge4Re](https://github.com/user-attachments/assets/63047221-78d1-4a9f-9d51-c8baf9085b96)
![dreamseeker_KO4XbQzoPn](https://github.com/user-attachments/assets/b53138de-fb71-4649-9765-f88b11040d60)

jwc1015 committed:
Merge branch 'Baystation12:dev' into bee

jwc1015 committed:
Updated lefthand.dmi to fix issue

Turns out two of the sprites were in the wrong place in the file, so I swapped them and it should work now.

jwc1015 commented:
> With the bee in your left hand, the left and right facing sprites are incorrect.
> 
> ![dreamseeker_97Vzmge4Re](https://private-user-images.githubusercontent.com/11140088/414537959-63047221-78d1-4a9f-9d51-c8baf9085b96.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDAwMDc2NDEsIm5iZiI6MTc0MDAwNzM0MSwicGF0aCI6Ii8xMTE0MDA4OC80MTQ1Mzc5NTktNjMwNDcyMjEtNzhkMS00YTlmLTlkNTEtYzhiYWY5MDg1Yjk2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjE5VDIzMjIyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY0ZTQxMzQzMDQxNzgzMGRhMTM2YjMwMWRmMThhZTY4MTg2ZTMyOTdlZDhmNThkYjkyNjVkMTMyMzU4YmU0NzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.4PJkQCiuRi9TG-dYXh-XhWRPEGXaL5dQ67JFg0D1DDk) ![dreamseeker_KO4XbQzoPn](https://private-user-images.githubusercontent.com/11140088/414537971-b53138de-fb71-4649-9765-f88b11040d60.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDAwMDc2NDEsIm5iZiI6MTc0MDAwNzM0MSwicGF0aCI6Ii8xMTE0MDA4OC80MTQ1Mzc5NzEtYjUzMTM4ZGUtZmI3MS00NjQ5LTk3NjUtZjg4YjExMDQwZDYwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjE5VDIzMjIyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU4NTc4YmMxOWM5MTQxMzZlYmFmZTc0MDdhMzY5ZTFkNjZmMzUwNGE5ZTZkNjQ3MTBmM2IwYzYwNjFiMjdjN2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.3w6vdQqDYdCZMt-4FrBImkNTLRMVS1xEEl1rBa6XrM4)

Sorry, I ended up accidentally closing this in the process of updating without a bunch of commits as I did things in the wrong order. I've moved the sprites to the correct positions and it should hopefully work now!
